appveyor.yml

```diff
@@ -25,6 +24,17 @@ test:
     - '**\Sarif.Driver.UnitTests.dll'
     - '**\Sarif.FunctionalTests.dll'
     - '**\Sarif.ValidationTests.dll'
+    
+artifacts:
+  - path: 'bld\bin'
```

Comment:
I wanted that (to drop the bld folder.) As Rob stated, the next steps for the build are to update the assemblyinfo versions and get the nuget package to build and deploy.

---
In reply to: [67393231](https://github.com/Microsoft/sarif-sdk/pull/364#discussion_r67393231) [](ancestors = 67393231)